ajax/libs/flatpickr/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "flatpickr",
+  "filename": "flatpickr.min.js",
+  "version": "1.8.0",
+  "description": "A lightweight, powerful datetimepicker",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/chmln/flatpickr.git"
+  },
+  "author": "Gregory",
+  "license": "MIT",
+  "homepage": "https://chmln.github.io/flatpickr",
+  "keywords": [
+    "javascript",
+    "datetimepicker",
+    "calendar",
+    "date",
+    "time",
+    "picker",
+    "lightweight"
+  ],
+  "lecense": "MIT",
+  "npmName": "flatpickr",
+  "npmFileMap": [
+    {
+      "basePath": "src",
+      "files": [
+        "flatpickr*"
+      ]
+    },
```

Comment:
Maybe we don't need to add the ```flatpickr.js```.
Because there is a ```flatpickr.min.js``` in dist file.
So could I just use the dist?
Thanks.